_gsocproposals/2019/proposal_ActsAlign.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Improving the precision of algorithms to hunt particles
+layout: gsoc_proposal
+project: Acts, IRIS-HEP
+year: 2019
+organization:
+- CERN
+- UC Berkeley
```

Comment:
Could you please add an organization description for `UC Berkeley`?

Reply:
Done. I changed it to UCB to match the rest.